src/main/java/uk/gov/justice/probation/courtcaseservice/controller/CourtCaseController.java

```diff
@@ -93,4 +93,20 @@ CaseListResponse getCaseList(@PathVariable String courtCode,
                 .collect(Collectors.toList());
         return CaseListResponse.builder().cases(courtCaseResponses).build();
     }
+
+    @ApiOperation(value = "For the date / cases passed, delete any cases NOT in the list.")
+    @ApiResponses(
+        value = {
+            @ApiResponse(code = 204, message = "OK", response = CaseListResponse.class),
+            @ApiResponse(code = 400, message = "Invalid request", response = ErrorResponse.class),
+            @ApiResponse(code = 401, message = "Unauthorised", response = ErrorResponse.class),
+            @ApiResponse(code = 403, message = "Forbidden", response = ErrorResponse.class),
+            @ApiResponse(code = 404, message = "If the court is not found by the code passed."),
+            @ApiResponse(code = 500, message = "Unrecoverable error whilst processing request.", response = ErrorResponse.class)
+        })
+    @ResponseStatus(value = HttpStatus.NO_CONTENT)
+    @PutMapping(value = "/court/{courtCode}/cases/purge", consumes = APPLICATION_JSON_VALUE, produces = APPLICATION_JSON_VALUE)
```

Comment:
Just wondering whether `purgeAbsent/missing` would be more a more descriptive name? Honestly not sure as it may raise more questions than it answers. The swagger description is very clear which is the main thing - up to you if you think this makes it clearer or not.

Reply:
It's a good point. Yes - earlier I was thinking along those lines because I named the service method "deleteMissingCases". I'll standardise around your "absent". Kept deleteAbsentCases in service and repo because the "delete" matches what happens in repo.